utils/api/definitions/enums.ts

```diff
@@ -1,3 +1,10 @@
+export enum BenefitKey {
+  oas = 'oas',
+  gis = 'gis',
+  alw = 'alw',
```

Comment:
alw or allowance ?

Reply:
alw is the short form // abbreviation we're using for allowance through the app! it's actually used by the gov in some internal documents I saw as well (which is actually what prompted me to use alw in the first place)